mmtrack/datasets/pipelines/processing.py

```diff
@@ -261,6 +261,125 @@ def __call__(self, pair_video_infos):
         return results
 
 
+@PIPELINES.register_module()
+class PairSampling(object):
```

Comment:
The code of the pipeline is good to merge.
1. However, we need firstly merge the pr of supportting ceph, since this pr contains some repeated codes.
2. Besides, we also need to align the performance of siamrpn++ on large dataset such as lasot, trackingnet, and got10k, before merging the pr.